lemminx-maven/src/main/java/org/eclipse/lemminx/maven/VersionValidator.java

```diff
@@ -12,34 +12,24 @@
 import org.apache.maven.artifact.DefaultArtifact;
 import org.apache.maven.artifact.handler.DefaultArtifactHandler;
 import org.apache.maven.model.Dependency;
-import org.eclipse.lemminx.dom.DOMDocument;
 import org.eclipse.lemminx.dom.DOMNode;
 import org.eclipse.lsp4j.Diagnostic;
 import org.eclipse.lsp4j.DiagnosticSeverity;
-import org.eclipse.lsp4j.Range;
 
 public class VersionValidator {
 
 	public static Diagnostic validateVersion(DiagnosticRequest diagnosticRequest) {
```

Comment:
This could also be an `Optional<Diagnostic>`, or even an `Optional<List<Diagnostic>>` so this could be referenced in MavenDiagnosticParticipant.tagDiagnostics.